docs/monitoring.md

```diff
@@ -0,0 +1,54 @@
+# Network Monitoring
```

Comment:
Is this document meant to be included in the packaged OTP release? It looks to me no so ok.

If yes:
* File needs to be made location agnostic, e.g. without `../apps` links.
* rebar.config needs to be updated - with copy of file

Reply:
updated